apps/registration/index.coffee

```diff
@@ -1,9 +0,0 @@
-express = require 'express'
-routes = require './routes'
-
-app = module.exports = express()
-
-app.set 'views', "#{__dirname}/templates"
-app.set 'view engine', 'jade'
-
-app.get '/register/:token', routes.acceptInvitation, routes.invalidInvitation
```

Comment:
Yeah. However before it was basically a security bug: you could sign up with one email then change it to an arbitrary one because we never "reconfirmed it" with this PR we enforce reconfirmations.